Microsoft.Toolkit.Win32/Microsoft.Toolkit.Win32.UI.Controls/Interop/WinRT/WebViewControlPermissionType.cs

```diff
@@ -41,11 +41,5 @@ public enum WebViewControlPermissionType
         /// Permission is for screen.
         /// </summary>
         Screen,
-
```

Comment:
Just making sure, this (among other things) was added in #2308 and now it's getting removed, is that right, or is there a commit conflict happening somewhere

Reply:
It was reverted in 8daf1adb and kept after merging to main by mistake. Reviewing other items to ensure everything is what it's supposed to be

Reply:
Revert is reverted in 87b5226b3811e6870080a0c4507602749fd0976c